src/main/java/org/apache/commons/crypto/stream/CipherInputStream.java

```diff
@@ -430,6 +429,42 @@ protected Cipher getCipher() {
   }
 
   /**
+   * Gets the IV.
+   *
+   * @return the IV.
+   */
+  public byte[] getIv() {
+    return iv;
+  }
+
+  /**
+   * Set the IV.
+   *
+   * @param iv the IV.
+   */
+  public void setIv(byte[] iv) {
```

Comment:
Do we really need setIv method? it is passed in constructor and we may don't need to change it.